app/components/ui/Inputs/Validation/ProductInput.tsx

```diff
@@ -57,7 +57,7 @@ export const ProductInput = React.forwardRef<HTMLInputElement, InputFormProps>(f
     subTitle: {
       requiredMessage: "This field is required",
       pattern: {
-        value: /^[-()#%/a-zA-Z0-9 ]{0,600}$/,
+        value: /^[-()#%/a-zA-Z0-9\n ]{0,600}$/,
```

Comment:
I know that `\n` needed for `shift+enter` in description but its better when you do it in separate commit with descriotion what problem it solve